specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -208,43 +252,57 @@ int main()
     return 0;
 }
 ```
+
 Process B (Register the inproc COM server and handle the background activation):
+
 ```cpp
 int main()
 {
-    PushNotificationActivationInfo info(
-        PushNotificationRegistrationKind::ComActivator,
-        winrt::guid("BACCFA91-F1DE-4CA2-B80E-90BE66934EC6"));
+    PushNotificationActivationInfo info = nullptr;
+    if (PushNotificationManager::IsActivatorSupported(PushNotificationRegistrationOptions::ComActivator))
+    {
+        // Register the ComActivator guid
+        info = PushNotificationActivationInfo(PushNotificationRegistrationOptions::ComActivator,
+            winrt::guid("BACCFA91-F1DE-4CA2-B80E-90BE66934EC6"));
+    }
+    else
+    {
+        return ERROR_NOT_SUPPORTED;
```

Comment:
Why not E_NOTIMPL? (the typical HRESULT used instead of Win32's ERROR_NOT_SUPPORTED)